ironfish/src/rpc/routes/mining/exportMined.ts

```diff
@@ -59,64 +61,34 @@ router.register<typeof ExportMinedStreamRequestSchema, ExportMinedStreamResponse
     Assert.isNotNull(node.chain.head, 'head')
     Assert.isNotNull(node.chain.latest, 'latest')
 
-    const scanForks = request.data?.forks == null ? false : true
+    const blockHash = request.data?.blockHash == null ? undefined : request.data.blockHash
```

Comment:
nit: Does nullish coalescing work? 
```suggestion
    const blockHash = request.data?.blockHash ?? undefined
```